week-3/task1/EventEmitterTest.js

```diff
@@ -0,0 +1,54 @@
+import EventEmitter from './EventEmitter.js';
```

Comment:
Since what want to achieve is basically a replica of the actual EventEmitter, I would recommend you to switch to the Node.js EventEmitter core implementation so you can easily identify the differences.
```ts
// import EventEmitter from './EventEmitter.js';
import EventEmitter from 'node:events';
```